iosHackthonApp/Modules/Shared/Components/Sources/Components/BottomSheet.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  BottomSheet.swift
+//  
+//
+//  Created by Gareth Miller on 26/11/2020.
+//
+
+import Foundation
+import SwiftUI
+import Theming
+
+fileprivate enum Constants {
+    static let cornerRadius: CGFloat = 16
+    static let indicatorWidth: CGFloat = 60
+    static let indicatorHeight: CGFloat = 6
+    static let minHeightRatio: CGFloat = 0.15
+}
+
+public struct BottomSheetView<Content: View>: View {
+
+    private let maxHeight: CGFloat
+    private let minHeight: CGFloat
+    private let content: Content
+    private let labelText: String
+
+    @GestureState private var translation: CGFloat = 0
+    @Binding private var isOpen: Bool
+
+    public init(isOpen: Binding<Bool>, maxHeight: CGFloat, labelText: String, @ViewBuilder content: () -> Content) {
+        self.minHeight = min(maxHeight * Constants.minHeightRatio, 50)
```

Comment:
Been a bit annoying finding a value that works for all devices here tbh.
This now works though, if the maxHeight * 0.15 > 50 (which it is on the larger devices) then we use 50 for the minHeight.
This stops the first item in the list being visible on the bottom sheet before it is opened.